services/metadata/src/metadata/routes/templates.clj

```diff
@@ -24,13 +24,27 @@
       :description "This endpoint returns the details of a single metadata attribute."
       (ok (templates/view-attribute attr-id)))
 
-    (GET* "/:template-id" []
+    (context* "/:template-id" []
       :path-params [template-id :- TemplateIdPathParam]
-      :query [params StandardUserQueryParams]
-      :return MetadataTemplate
-      :summary "View a Metadata Template"
-      :description "This endpoint returns the details of a single metadata template."
-      (ok (templates/view-template template-id)))))
+
+      (GET* "/" []
+        :query [params StandardUserQueryParams]
+        :return MetadataTemplate
+        :summary "View a Metadata Template"
+        :description "This endpoint returns the details of a single metadata template."
+        (ok (templates/view-template template-id)))
+
+      (GET* "/blank-csv" []
+        :query [{:keys [attachment]} CSVDownloadQueryParams]
+        :summary "Get a blank CSV template file for a metadata template."
+        :description "This endpoint returns a CSV file suitable for a specific template, ready to be filled in with specific values. It's intended to be downloaded to be filled out by the user, then reuploaded for use with the bulk metadata endpoints."
```

Comment:
Minor recommendations:
These docs could use line-breaks to help readability in the code.
I think the 2nd `to be` could be replaced with a comma: `intended to be downloaded to be filled out by the user, ...` -> `intended to be downloaded, filled out by the user, ...`